tests/telemetry/test_writer.py

```diff
@@ -70,61 +70,65 @@ def test_app_started_event(telemetry_writer, test_agent_session, mock_time):
     events[0]["payload"]["configuration"].sort(key=lambda c: c["name"])
 
     payload = {
-        "configuration": [
```

Comment:
This diff is messy. I added sorting to the data since order doesn't matter so future updates will be easier to the data.

The changes here are the omission of the old telemetry entries  `DD_TRACE_SAMPLE_RATE` and `DD_LOGS_INJECTION` and addition of the new entries `trace_sample_rate`, `logs_injection_enabled` and `trace_header_tags` at the bottom.